db/schema.rb

```diff
@@ -38,6 +39,28 @@
     t.datetime "device_updated_at", null: false
   end
 
+  create_table "facilities", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
This is a nicer approach than the `before_create` callback we used for protocols. We'll change that.